### ISSUE TITLE
how to change yAxis position?

### ISSUE THREAD

**Issue by khm0651** [Kotlin/kandy#433]
i saw https://github.com/Kotlin/kandy/issues/306  
but i cant find change yAxis position left or right  in API doc
i want change yAxis position left to right how can i do that??

like that python matplotlib 
yaxis.tick_right()

**Comment by AndreiKingsley** [comment]
Hi! You can do it with axis position setting (axis settings):
```
plot {
   // first way - in mapping configuration
   points {
        x(...) 
        y(...) {
            axis.position = AxisPosition.OPPOSITE
        }
    }
    // second way - free axis configuration
   y.axis.position = AxisPosition.OPPOSITE
}
```

But I just looked, it's not working right now because of a bug #434. Will fix it soon.

**Comment by khm0651** [comment]
@AndreiKingsley thank you so much

**Comment by AndreiKingsley** [comment]
Fixed in 0.7.1.
